### ISSUE TITLE
【PP-YOLOE+ vs YOLOv6 Update】官方精度数据更新 / Update of the Official Performance Comparison

### ISSUE THREAD

**Issue by nemonameless** [meituan/YOLOv6#431]
English Version

Hi Meituan YOLOv6 team (@Chilicyy , @mtjhl ),

It is glad to see that YOLOv6 has improved the accuracy and speed again. It is also a pleasure to see that the [PP-YOLOE](https://github.com/PaddlePaddle/PaddleDetection/tree/develop/configs/ppyoloe) series algorithm developed by [PaddleDetection](https://github.com/PaddlePaddle/PaddleDetection) has been added to the comparison.
Here we would like to make a supplementary explanation for the accuracy of  PP-YOLOE:

The latest version of PP-YOLOE is [PP-YOLOE+](https://github.com/PaddlePaddle/PaddleDetection/tree/release/2.5/configs/ppyoloe), which was released together with PaddleDetection [release/2.5] on August 26, 2022. In fact, the training was completed in early August. For details, please refer to [PP-YOLOE+  ModelZoo](https://github.com/PaddlePaddle/PaddleDetection/tree/release/2.5/configs/ppyoloe#model-zoo) and [mAP/FPS comparison table](https://github.com/PaddlePaddle/PaddleDetection/blob/release/2.5/docs/images/ppyoloe_plus_map_fps.png).

|     Model      | Epoch | GPU number | images/GPU |  backbone  | input shape | Box AP<sup>val<br>0.5:0.95 | Box AP<sup>test<br>0.5:0.95 | Params(M) | FLOPs(G) | V100 FP32(FPS) | V100 TensorRT FP16(FPS) |                                       download                                       |                   config                    |
|:--------------:|:-----:|:-------:|:----------:|:----------:| :-------:|:--------------------------:|:---------------------------:|:---------:|:--------:|:---------------:| :---------------------: |:------------------------------------------------------------------------------------:|:-------------------------------------------:|
|  PP-YOLOE+_s   |  80   |     8      |     8      | cspresnet-s |     640     |            43.7            |            43.9             |   7.93    |  17.36   |   208.3   |  333.3   | [model](https://paddledet.bj.bcebos.com/models/ppyoloe_plus_crn_s_80e_coco.pdparams) | [config](./ppyoloe_plus_crn_s_80e_coco.yml) |
|  PP-YOLOE+_m   |  80   |     8      |     8      | cspresnet-m |     640     |            49.8            |            50.0             |   23.43   |  49.91   |   123.4   |  208.3   | [model](https://paddledet.bj.bcebos.com/models/ppyoloe_plus_crn_m_80e_coco.pdparams) | [config](./ppyoloe_plus_crn_m_80e_coco.yml) |
|  PP-YOLOE+_l   |  80   |     8      |     8      | cspresnet-l |     640     |            52.9            |            53.3             |   52.20   |  110.07  |   78.1    |  149.2   | [model](https://paddledet.bj.bcebos.com/models/ppyoloe_plus_crn_l_80e_coco.pdparams) | [config](./ppyoloe_plus_crn_l_80e_coco.yml) |
|  PP-YOLOE+_x   |  80   |     8      |     8      | cspresnet-x |     640     |            54.7            |            54.9             |   98.42   |  206.59  |   45.0    |   95.2   | [model](https://paddledet.bj.bcebos.com/models/ppyoloe_plus_crn_x_80e_coco.pdparams) | [config](./ppyoloe_plus_crn_x_80e_coco.yml) |

We are looking forward to the release of your team's paper,  and also hope that [PP-YOLOE+](https://github.com/PaddlePaddle/PaddleDetection/tree/release/2.5/configs/ppyoloe) could be added to the comparison table of the paper to make the comparison of YOLO family more comprehensive.

Later, [PaddleDetection](https://github.com/PaddlePaddle/PaddleDetection) team will also pay more attention to the end to end performance of detection algorithm, and continue to improve the accessibility and applicability of algorithms based on industrial practices. Follow our updates if you are interested. 😊

Best,
[PaddleDetection](https://github.com/PaddlePaddle/PaddleDetection) team

================================================================================

Chinese Version 中文版

尊敬的美团YOLOv6团队 (@Chilicyy , @mtjhl )，你们好！

非常高兴看到YOLOv6再度提升了精度速度以及发布了大模型，也非常荣幸看到飞桨PaddleDetection开发的PP-YOLOE系列算法被加入到了对比当中。其中对于PP-YOLOE的精度指标想做下补充说明：

目前PP-YOLOE最新版是[PP-YOLOE+](https://github.com/PaddlePaddle/PaddleDetection/tree/release/2.5/configs/ppyoloe)，于2022年8月26日随PaddleDetection [release/2.5](https://github.com/PaddlePaddle/PaddleDetection/tree/release/2.5)版本一起发布，是8月初训完的，详情可以参考[PP-YOLOE+ ModelZoo](https://github.com/PaddlePaddle/PaddleDetection/tree/release/2.5/configs/ppyoloe#model-zoo)和[mAP和FPS对比表格](https://github.com/PaddlePaddle/PaddleDetection/blob/release/2.5/docs/images/ppyoloe_plus_map_fps.png)。

|       模型        | Epoch |   GPU个数   | 每GPU图片个数 |  骨干网络  |    输入尺寸    | Box AP<sup>val<br>0.5:0.95 | Box AP<sup>test<br>0.5:0.95 | Params(M) | FLOPs(G) | V100 FP32(FPS) | V100 TensorRT FP16(FPS) |                                         模型下载                                         |                    配置文件                     |
|:---------------:|:-----:|:---------:|:--------:|:----------:|:----------:|:--------------------------:|:---------------------------:|:---------:|:--------:|:---------------:| :---------------------: |:------------------------------------------------------------------------------------:|:-------------------------------------------:|
|   PP-YOLOE+_s   |  80   |     8     |    8     | cspresnet-s |    640     |            43.7            |            43.9             |   7.93    |  17.36   |   208.3   |  333.3   | [model](https://paddledet.bj.bcebos.com/models/ppyoloe_plus_crn_s_80e_coco.pdparams) | [config](https://github.com/PaddlePaddle/PaddleDetection/tree/release/2.5/configs/ppyoloe/ppyoloe_plus_crn_s_80e_coco.yml) |
|   PP-YOLOE+_m   |  80   |     8     |    8     | cspresnet-m |    640     |            49.8            |            50.0             |   23.43   |  49.91   |   123.4   |  208.3   | [model](https://paddledet.bj.bcebos.com/models/ppyoloe_plus_crn_m_80e_coco.pdparams) | [config](https://github.com/PaddlePaddle/PaddleDetection/tree/release/2.5/configs/ppyoloe/ppyoloe_plus_crn_m_80e_coco.yml) |
|   PP-YOLOE+_l   |  80   |     8     |    8     | cspresnet-l |    640     |            52.9            |            53.3             |   52.20   |  110.07  |   78.1    |  149.2   | [model](https://paddledet.bj.bcebos.com/models/ppyoloe_plus_crn_l_80e_coco.pdparams) | [config](https://github.com/PaddlePaddle/PaddleDetection/tree/release/2.5/configs/ppyoloe/ppyoloe_plus_crn_l_80e_coco.yml) |
|   PP-YOLOE+_x   |  80   |     8     |    8     | cspresnet-x |    640     |            54.7            |            54.9             |   98.42   |  206.59  |   45.0    |   95.2   | [model](https://paddledet.bj.bcebos.com/models/ppyoloe_plus_crn_x_80e_coco.pdparams) | [config](https://github.com/PaddlePaddle/PaddleDetection/tree/release/2.5/configs/ppyoloe/ppyoloe_plus_crn_x_80e_coco.yml) |

我们非常期待贵团队的论文的发布，同时也希望[PP-YOLOE+](https://github.com/PaddlePaddle/PaddleDetection/tree/release/2.5/configs/ppyoloe)可以加入到论文的对比表格中，使得YOLO家族的对比更加全面。

后续飞桨研发团队也会进一步关注业界检测算法端到端性能，并基于产业实践持续优化算法易用性与泛化性，以更好的服务于检测算法的产业落地，敬请关注。😊

飞桨PaddleDetection团队

**Comment by Chilicyy** [comment]
Thanks for your attention!  For fair comparision, our models in the comparison table of technical report are trained for 300 epochs without pre-training or any external data. More details can be found in our [technical report](https://arxiv.org/pdf/2209.02976.pdf). Thanks.
